lib/ansible/plugins/inventory/docker_machine.py

```diff
@@ -90,16 +92,16 @@ class InventoryModule(BaseInventoryPlugin, Constructable, Cacheable):
     NAME = 'docker_machine'
 
     def _run_command(self, args):
-        command = ['docker-machine']
+        command = [(shutil.which('docker-machine'))]
```

Comment:
Don't use that, use the Ansible function (see my other comment). Also, just do that once and reuse the result.